.circleci/config.yml

```diff
@@ -394,6 +394,7 @@ jobs:
       DDAGENT_HOSTNAME: 127.0.0.1
       DD_AGENT_HOST: 127.0.0.1
       HTTPBIN_HOSTNAME: httpbin_integration
+      DATADOG_HAVE_DEV_ENV: 1
```

Comment:
Should everything run in CircleCI set `DATADOG_HAVE_DEV_ENV`?

Reply:
Ideally, yes. Do you know how to set it across the board?

Reply:
Most CIs do have this capability. In CircleCI's case, it seems like we'd [have to use the UI](https://circleci.com/docs/2.0/env-vars/)?

Reply:
Thanks @morrisonlevi. I'll merge this as-is for now to make sure it gets into the next release, but I'll look into adding this env var across the board from the UI.

Reply:
@morrisonlevi I've added `DATADOG_HAVE_DEV_ENV=1` via the UI but I have a concern: We have cleared out the CircleCI env vars in the past for various reasons so I'm a bit concerned that at one point we may inadvertently remove `DATADOG_HAVE_DEV_ENV` from the env vars and it will silently skip over a lot of crucial tests. So I'm thinking it might be better to keep this env var set explicitly in the CI config to reduce the chances of accidentally skipping over tests.

Alternatively we could add a job that checks that `DATADOG_HAVE_DEV_ENV == 1` and will fail if it is not set. That way if the env var is removed from the UI, at least that test will fail.

I don't have any strong opinions either way. WDYT?

Reply:
Avoiding the UI seems fine to me; that's hidden state that seems like it could bite us.